airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/manifest.yaml

```diff
@@ -24,6 +24,24 @@ definitions:
             q: search
           request_headers:
             Linkedin-Version: "202404"
+          error_handler:
+            type: CompositeErrorHandler
+            error_handlers:
+              - type: DefaultErrorHandler
+                response_filters:
+                  - type: HttpResponseFilter
+                    action: RETRY
+                    error_message_contains: "Temporary failure in name resolution"
+                    predicate: >-
+                      {{ "Temporary failure in name resolution" in
+                      response }}
```

Comment:
Looks really good, @agarctfi . Just one question...

This is probably just a limit in my understanding. Are these 2 conditions combined or additive? Either or both look like they could catch the issue, but I'm not very familiar with this API personally so I'm not sure if one or the other is preferred, or if setting both is preferred.